text/0016-named-tracers.md

```diff
@@ -12,43 +12,53 @@ _Creating Tracers and Meters using a factory mechanism and naming those Tracers
 
 ## Motivation
 
-The mechanism of "Named Tracers and Meters" proposed here is motivated by following scenarios:
+The mechanism of "Named Tracers and Meters" proposed here is motivated by the following scenarios:
 
-* For a consumer of OpenTelemetry instrumentation libraries, there is currently no possibility of influencing the amount of the data produced by such libraries. Instrumentation libraries can easily "spam" backend systems, deliver bogus data or - in the worst case - crash or slow down applications. These problems might even occur suddenly in production environments caused by external factors such as increasing load or unexpected input data.
+### Faulty or expensive instrumentation
 
-* If a library hasn't implemented [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md) correctly or those conventions change over time, it's currently hard to interpret and sanitize these data selectively. The produced Spans or Metrics cannot be associated with those instrumentation libraries later.
+For an operator of an application using OpenTelemetry, there is currently no way to influence the amount of data produced by reporting libraries. Reporting libraries can easily "spam" backend systems, deliver bogus data, or -- in the worst case -- crash or slow down applications. These problems might even occur suddenly in production environments because of external factors such as increasing load or unexpected input data.
+
+### Reporting library identification
+
+If a reporting library hasn't implemented [semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-semantic-conventions.md) correctly or those conventions change over time, it's currently hard to interpret and sanitize data produced by it selectively. The produced Spans or Metrics cannot later be associated with the library which reported them, either in the processing pipeline or the backend.
+
+### Disable instrumentation of pre-instrumented libraries
+
+It is the eventual goal of OpenTelemetry that library vendors implement the OpenTelemetry API, obviating the need to auto-instrument their library. An operator should be able to disable the telemetry that is built into some database driver or other library and provide their own integration if the built-in telemetry is lacking in some way. This should be possible even if the developer of that database driver has not provided a configuration to disable telemetry.
+
+## Solution
```

Comment:
I think when conforming to the [OTEP template](https://github.com/open-telemetry/oteps/blob/master/0000-template.md), this would be the "Explanation" but I wonder if we can find a good sub-heading for our current explanation, and whether our current "Internal details" vs "Explanation" division makes sense. This now sounds like a can of worms, so "Solution" should actually be fine. 😄